example/server/index.ts

```diff
@@ -519,9 +519,33 @@ app.post('/payment-sheet', async (_, res) => {
     { apiVersion: '2020-08-27' }
   );
   const paymentIntent = await stripe.paymentIntents.create({
-    amount: 1099,
+    amount: 5099,
     currency: 'usd',
     customer: customer.id,
+    shipping: {
+      name: 'Jane Doe',
+      address: {
+        state: 'Texas',
+        city: 'Houston',
+        line1: '1459  Circle Drive',
+        postal_code: '77063',
+        country: 'US',
+      },
+    },
+    // Edit the following to support different payment methods in your PaymentSheet
```

Comment:
Interesting, looks like this is user-facing.  I wonder why we aren't using the example server(s) the iOS and Android platforms use (you can see them here: https://glitch.com/@stripe-mobile-payments)

Reply:
good question! I'm honestly not sure why this approach was opted for instead...on CI, we rely on our glitch server rather than the local one here. May have just been added before glitch was considered and is still around. Suppose we could probably get rid of this server altogether from the repo